include/libtorrent/kademlia/dht_tracker.hpp

```diff
@@ -155,6 +156,7 @@ namespace libtorrent { namespace dht
 		bdecode_node m_msg;
 
 		counters& m_counters;
+		dht_storage_interface* m_storage;
```

Comment:
a part of me feels like it would be more appropriate to make this a reference. a reference cannot be rebound and cannot be null, which I think fits the semantics of this pointer.